provider/provider_test.go

```diff
@@ -142,61 +141,3 @@ func test(t *testing.T, dir string, opts ...providertest.Option) *providertest.P
 	)
 	return providertest.NewProviderTest(dir, opts...)
 }
-
```

Comment:
This retrieved the dependencies of the current version, but we need the deps of the baseline version